2017/qualification round/OversizedPancakeFlipper/src/main/java/train/google/code/jam/PancakeFlipper.java

```diff
@@ -1,14 +1,50 @@
 package train.google.code.jam;
 
 public class PancakeFlipper {
+
+    private static final char WHITE_PANCAKE = '-';
+    private static final char HAPPY_PANCAKE = '+';
+
     private final Integer size;
 
     public PancakeFlipper(Integer size) {
         this.size = size;
     }
 
-    public Integer flip_to_perfect_happy(String currentPancake) {
-        return null;
+    public String flip(String pancake) {
```

Comment:
I would say to change it as rowOfpancakes.
pancake - one pancake
row of pancake - group of pancakes.

Reply:
no entiendo, no hay filas de pancakes, como mucho columnas.

Reply:
y sobre lo del pancakehouse debería controlar todo ... yo creo que el pancakehouse gestionará a los componentes que tiene, pero cada componente debería tener su propia lógica y responsabilidad